src/components/Calculator.js

```diff
@@ -1,35 +1,95 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
 import PropTypes from 'prop-types';
+import calculate from '../logic/calculate';
+
+const Display = ({ inputs, whoChanged }) => {
+  let finaloutput = '';
+  if (!inputs.total && !inputs.operation && !inputs.next) finaloutput = '0';
+  else if (whoChanged.dot) {
+    if (!inputs.operation) finaloutput = inputs.next;
+    else {
+      finaloutput = !inputs.operation ? inputs.next : `${inputs.total || ''} ${inputs.operation || ''} ${inputs.next || ''}`;
+    }
+  } else {
+    finaloutput = `${inputs.total || ''} ${inputs.operation || ''} ${inputs.next || ''}`;
+  }
+  return (
+    <div className="span4col griditem2 weirdgray">
+      {finaloutput}
+      {/* {whoChanged.total.toString()}
+      {whoChanged.operation.toString()}
+      {whoChanged.next.toString()} */}
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 👍🏾